projects/snapshot_bot/snapshot_bot.cc

```diff
@@ -394,7 +394,7 @@ class RobotFSM : FSM<State>::StateHandler
                     const auto &attitude = pose.attitude();
 
                     // Write extra logging data
-                    m_LogFile << ", " << objectData.getFrameNumber() << ", " << position[0] << ", " << position[1] << ", " << position[2] << ", " << attitude[0] << ", " << attitude[1] << ", " << attitude[2];
+                    m_LogFile << ", " << objectData.getFrameNumber() << ", " << position[0].value() << ", " << position[1].value() << ", " << position[2].value() << ", " << attitude[0].value() << ", " << attitude[1].value() << ", " << attitude[2].value();
```

Comment:
I feel this could probably use a line break somewhere :laughing:

Reply:
Fine point